### PR TITLE
CI: Move RuntimeValidatedPolicyDropAllTests to RuntimeValidatedPolicies

### PR DESCRIPTION
**Pull request by manalibhutiyani**
Make the DROP_ALL gingko test more time-efficient and resource effecient,
by grouping it with RuntimeValidatedPolicies

Fixes: #3880
Signed-Off-By: Manali Bhutiyani <manali@covalent.io>

### CODE REVIEW

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
@eloycoto @ianvernon Followup of the DROP_ALL tests review comments.

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-me-please

File: test/runtime/Policies.go [hunk 1]
```diff
@@ -1010,6 +1013,78 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 		})
 	})
+	Context("DROP_ALL  Policy test", func() {
+		AfterEach(func() {
+			vm.ContainerRm(dropAllContainer).ExpectSuccess("Container dropAllContainer cannot be deleted")
+		})
+
+		It("DROP_ALL Ingress Policy test", func() {
+			res := vm.ContainerCreate(dropAllContainer, "tgraf/netperf", helpers.CiliumDockerNetwork, "", "")
```
Comment (eloycoto) [inline test/runtime/Policies.go]: could you change `tgraf/netperf` for `helpers.NetperfImage` ?

File: test/runtime/Policies.go [hunk 2]
```diff
@@ -1010,6 +1013,78 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
...
+			ctx, cancel := context.WithCancel(context.Background())
+			dropRes := vm.ExecContext(ctx, "cilium monitor --type drop")
+			defer cancel()
+
+			endpoints, err := vm.GetAllEndpointsIds()
+			Expect(err).Should(BeNil(), "Unable to get IDs of endpoints")
+			endpointID, exists := endpoints[dropAllContainer]
+			Expect(exists).To(BeTrue(), "Expected endpoint ID to exist for %s", dropAllContainer)
+			ingressEpModel := vm.EndpointGet(endpointID)
+			Expect(ingressEpModel).To(Not(BeNil()), "nil model returned for endpoint %s", endpointID)
```
Comment (eloycoto) [inline test/runtime/Policies.go]:
```
Expect(ingressEpModel).NotTo(BeNil(), "nil model returned for endpoint %s", endpointID)
```

With NotTo I think that it's clear

File: test/runtime/Policies.go [hunk 3]
```diff
@@ -1010,6 +1013,78 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
...
+			res.ExpectFail("Unexpectedly able to ping endpoint with DROP_ALL ingress policy")
+
+			By("Testing cilium monitor drop")
+			err = dropRes.WaitUntilMatch("drop (Policy denied (L3))")
+			Expect(err).To(BeNil(), "DROP all on ingress failed.")
+		})
+
+		It("DROP_ALL Egress Policy test", func() {
+			By("Create an endpoint with no labels to test DROP_ALL")
+			res := vm.ContainerCreate(dropAllContainer, "tgraf/netperf", helpers.CiliumDockerNetwork, "", "ping 8.8.8.8")
```
Comment (eloycoto) [inline test/runtime/Policies.go]: change netperf with the const

File: test/runtime/Policies.go [hunk 4]
```diff
@@ -1010,6 +1013,78 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
...
+			ctx, cancel := context.WithCancel(context.Background())
+			dropRes := vm.ExecContext(ctx, "cilium monitor --type drop")
+			defer cancel()
+
+			endpoints, err := vm.GetAllEndpointsIds()
+			Expect(err).To(BeNil(), "Unable to get IDs of endpoints")
+			endpointID, exists := endpoints[dropAllContainer]
+			Expect(exists).To(BeTrue(), "Expected endpoint ID to exist for %s", dropAllContainer)
+			ingressEpModel := vm.EndpointGet(endpointID)
+			Expect(ingressEpModel).To(Not(BeNil()), "nil model returned for endpoint %s", endpointID)
```
Comment (eloycoto) [inline test/runtime/Policies.go]: NotTo()

**Review by eloycoto — COMMENTED** [review]
Minor nitpicks, but now I think that it's much better and simpler to understand. Thanks

File: test/runtime/Policies.go (hunk 4 above)
Comment (manalibhutiyani) [reply test/runtime/Policies.go]: done

File: test/runtime/Policies.go (hunk 3 above)
Comment (manalibhutiyani) [reply test/runtime/Policies.go]: done

File: test/runtime/Policies.go (hunk 2 above)
Comment (manalibhutiyani) [reply test/runtime/Policies.go]: done

File: test/runtime/Policies.go (hunk 1 above)
Comment (manalibhutiyani) [reply test/runtime/Policies.go]: done

**Comment by manalibhutiyani** [comment]
test-me-please

**Review by eloycoto — APPROVED** [review]

**Pull request merged**